style.css

```diff
@@ -0,0 +1,1878 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap");
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+
+  /* 1 */
+  border-width: 0;
+
+  /* 2 */
+  border-style: solid;
+
+  /* 2 */
+  border-color: #e5e7eb;
+
+  /* 2 */
+  --tw-border-spacing-x: 0;
+  --tw-border-spacing-y: 0;
+  --tw-translate-x: 0;
+  --tw-translate-y: 0;
+  --tw-rotate: 0;
+  --tw-skew-x: 0;
+  --tw-skew-y: 0;
+  --tw-scale-x: 1;
+  --tw-scale-y: 1;
+  --tw-scroll-snap-strictness: proximity;
+  --tw-ring-offset-width: 0;
+  --tw-ring-offset-color: #fff;
+  --tw-ring-color: rgb(59 130 246 / 50%);
+  --tw-ring-offset-shadow: 0 0 #0000;
+  --tw-ring-shadow: 0 0 #0000;
+  --tw-shadow: 0 0 #0000;
+  --tw-shadow-colored: 0 0 #0000;
+}
+
+::before,
+::after {
+  --tw-content: "";
+}
+
+html {
+  line-height: 1.5;
+  tab-size: 4;
+
+  /* 3 */
+  font-family:
+    lato,
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    Roboto,
+    "Helvetica Neue",
+    Arial,
+    "Noto Sans",
+    sans-serif,
+    "Apple Color Emoji",
+    "Segoe UI Emoji",
+    "Segoe UI Symbol",
+    "Noto Color Emoji";
+
+  /* 4 */
+  font-feature-settings: normal;
+
+  /* 5 */
+}
+
+/*
+1. Remove the margin in all browsers.
+2. Inherit line-height from `html` so users can set them as a class directly on the `html` element.
+*/
+
+body {
+  margin: 0;
+
+  /* 1 */
+  line-height: inherit;
+
+  /* 2 */
+  position: relative;
+}
+
+.header-top-inner,
+.navbar-inner,
+.container,
+.aboutcontainer {
+  width: 83.5%;
+}
+
+.container {
+  background: url("./public/images/header-bg.jpg");
+  margin-top: 22px;
+  z-index: -99;
+}
+
+.aboutcontainer {
+  background: url("./public/images/header-bg.jpg");
+  z-index: -999;
+}
+
+.stop-scrolling {
+  height: 100%;
+  overflow: hidden;
+}
+
+.back {
+  background: url("./public/images/header-text-bg.png");
+  font-size: 2px;
+  color: #ec5242;
+}
```

Comment:
- [ ] Kindly make sure to fix the background on the main title of the page. At this moment the back has covered the whole title, kindly make sure to fix that in order to meet the provided designs. 

   - **YOUR DESIGN** 
   
     ![image](https://user-images.githubusercontent.com/87197412/210707928-4064ebe9-4d7d-4fcf-b758-ba8ec0de3e24.png)

     ![image](https://user-images.githubusercontent.com/87197412/210709050-35d57d70-86e1-4871-903a-b8e0d6a2a339.png)


  - **PROVIDED DESIGN**
  
      ![image](https://user-images.githubusercontent.com/87197412/210708005-fefb2888-7fda-49f9-9819-3a8ca9c71d6d.png)

     ![image](https://user-images.githubusercontent.com/87197412/210708928-d1c315f2-8424-4160-bd26-96af193951ca.png)